modules/ossm-jaeger-supported-configurations.adoc

```diff
@@ -0,0 +1,45 @@
+////
+This REFERENCE module included in the following assemblies:
+-assembly.adoc
+////
+
+[id="ossm-jaeger-supported-configuration_{context}"]
+= Jaeger supported configuration
+
+For the {ProductVersion} release of {ProductName}the only supported configuration of Jaeger is as follows:
+
+.Supported Jaeger configurations
+[options="header"]
+[cols="l, a"]
+|====
+|Spec|Notes
+|strategy: production
+storage:
+    type: elasticsearch
+
+|Elasticsearch is the only supported storage option for this release.  And only when using the elasticsearch-operator to self-provision an Elasticsearch cluster.
+
+|strategy: allinone
+  storage: memory
+|The allinone strategy is available in this release but it is *NOT* supported for production.  This strategy should only be used for development, demo, and test environments due to the fact that data in memory will not be retained.
```

Comment:
Other options are available, as shown in the https://github.com/jaegertracing/jaeger-operator/blob/master/deploy/examples/all-in-one-with-options.yaml example - both the options under allinone and storage (e.g. setting the max traces).

Also shows configuring the UI - problem is that this would also require more documentation about the structure. So possibly UI config should be left for the standalone Jaeger.